modules/@ergonode/products-batch-actions/src/components/Modals/UpdateProductsModal.vue

```diff
@@ -60,46 +88,97 @@ export default {
             default: () => {},
         },
     },
+    data() {
+        return {
+            fetchingAttributes: {},
+            attributes: {},
+            formItems: [],
+        };
+    },
     computed: {
         title() {
             return this.$t('@ProductsBatchActions.productBatchAction.components.UpdateProductsModal.title', {
                 info: this.selectedProductsCount,
             });
         },
-        formItems() {
+        verticalTabs() {
             return [
                 {
-                    id: getUUID(),
-                    component: TextField,
-                    label: 'First',
+                    title: this.$t('@ProductsBatchActions.attribute._.title'),
+                    component: () => import('@ProductsBatchActions/extends/attribute/components/VerticalTabs/AttributesVerticalTab'),
```

Comment:
Keep all translation in one business area, because this module is the multi-business content wrapper. Not needed to separate this to extend directory.